addons/account_peppol/wizard/account_move_send_views.xml

```diff
@@ -7,6 +7,8 @@
             <field name="inherit_id" ref="account.account_move_send_form"/>
             <field name="arch" type="xml">
                 <xpath expr="//div[@name='warnings']" position="inside">
+                    <!-- the ubl_partner_warning field is there to prevent a traceback in case ubl_cii isn't up to date -->
+                    <div invisible="1"><field name="ubl_partner_warning"/></div>
```

Comment:
Only this part is necessary imo

Reply:
`peppol_warning` is hidden depending on whether `ubl_partner_warning` has some warning or not. 
Without the rest of the diff, if `ubl_partner_warning` is computed to something it won't be visible in the view, since the views are not updated. `peppol_warning` won't be displayed either, because  `ubl_partner_warning` has some value in it. Hence, neither warning will be visible.